### PR TITLE
New provider Person

### PR DESCRIPTION
**Pull request by agoalofalife**
Add new provider Person

### CODE REVIEW

**Comment by bxcodec** [comment]
Wow. Would you mind update the Readme too, about this new feature? 

* btw, thanks for the PR

**Comment by agoalofalife** [comment]
Yes , a little later

File: lorem_test.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package faker
+
+import (
+	"github.com/agoalofalife/faker/support/slice"
```
Comment (bxcodec) [inline lorem_test.go]: sorry, but change this to `bxcodec`

Comment (agoalofalife) [reply lorem_test.go]: @bxcodec done

**Comment by bxcodec** [comment]
@agoalofalife  LGTM

**Comment by agoalofalife** [comment]
Who will merge?

**Comment by bxcodec** [comment]
When I've approved it, you can merge it directly.  :)

**Comment by agoalofalife** [comment]
good

**Pull request merged**